src/js/core.js

```diff
@@ -957,8 +957,8 @@
             }
 
             // https://github.com/yabwe/medium-editor/issues/1496
-            // ensure the focus remains in the editor for Firefox
-            if (MediumEditor.util.isFF) {
+            // But only if something is selected i.e, getSelectedParentElement !== {}
+            if (this.getSelectedParentElement().length > 0 && MediumEditor.util.isFF) {
```

Comment:
Root cause, In some cases when `execCommand` was called in FF, it checked for Firefox but never checked if an element is selected or not! And Hence, the error 
```MediumEditor.util.getContainerEditorElement(this.getSelectedParentElement()).focus()``` .

Resolved.